### ISSUE TITLE
Add helpful information to the SETUP.md

### ISSUE THREAD

**Issue by kytrinyx** [exercism/xobjective-c#17]
The contents of the SETUP.md file gets included in
the README.md that gets delivered when a user runs the `exercism fetch`
command from their terminal.

At the very minimum, it should contain a link to the relevant
language-specific documentation on
[help.exercism.io](http://help.exercism.io).

It would also be useful to explain in a generic way how to run the tests.
Remember that this file will be included with _all_ the problems, so it gets
confusing if we refer to specific problems or files.

Some languages have very particular needs in terms of the solution: nested
directories, specific files, etc. If this is the case here, then it would be
useful to explain what is expected.

------

Thanks, @tejasbubane for suggesting that we add this documentation everywhere.
See [exercism.io#2198](https://github.com/exercism/exercism.io/issues/2198).

**Comment by brockboland** [comment]
I was looking at the SETUP file in some other language repos for an idea of how this is being used elsewhere, and I like [that the python version](https://github.com/exercism/xpython/blob/master/SETUP.md) is explicit about where the code files should live (at least in part because I made the mistake of putting my code elsewhere when I first got started). Seems to me that bit is worth including.

Beyond that, I wonder if linking to [the help page](http://help.exercism.io/getting-started-with-objective-c.html) is the best option—but then, I tend to be overly verbose in my documentation, so I'm leaning toward fleshing out that page with more detail, and don't want to simply duplicate it in both places.
